salt/states/sysctl.py

```diff
@@ -102,7 +103,11 @@ def present(name, value, config=None):
         return ret
 
     try:
-        update = __salt__["sysctl.persist"](name, value, config)
+        if ignore:
+            # ignore is a linux only sysctl setting
+            update = __salt__["sysctl.persist"](name, value, config, ignore)
```

Comment:
The `sysctl.persist` function in `salt/modules/linux_sysctl.py` does not have an `ignore` option. None of the `sysctl.persist` functions have an `ignore` option.

If you want to support ignore on Linux, it will need to be added.

Reply:
It used to ;) #55719

If PRs are being reviewed again, I can take a look at re-adding.